daemon/execdriver/windows/commandlinebuilder.go

```diff
@@ -0,0 +1,38 @@
+//+build windows
+
+package windows
+
+import (
+	"errors"
+	"syscall"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/docker/docker/daemon/execdriver"
+)
+
+// createCommandLine creates a command line from the Entrypoint and args
+// of the ProcessConfig. It escapes the arguments if they are not already
+// escaped
+func createCommandLine(processConfig *execdriver.ProcessConfig, escapeArgs bool) (commandLine string, err error) {
+	// While this should get caught earlier, just in case, validate that we
+	// have something to run.
+	if processConfig.Entrypoint == "" {
+		err := errors.New("No entrypoint specified")
+		logrus.Error(err)
```

Comment:
Ah, it was there already. But it shouldn't, also such logline will tell nothing to reader.